tests/test_download_photos.py

```diff
@@ -1485,7 +1484,7 @@ def test_delete_after_download_keep_recent_days(self) -> None:
                             "--threads-num",
                             "1",
                             "--delete-after-download",
-                            "--keep-recent-days",
+                            "--keep-icloud-recent-days",
                             "5",
```

Comment:
It may be more valuable to check edge of the interval instead of definate overlapping: Running on 8/2/2018 with `--keep-icloud-recent-days 2` should keep img from 7/31/2018, but running on 8/2/2018 with `--keep-icloud-recent-days 1` should not keep img from 7/31/2018, right?

Running on 7/31/2018 with `--keep-icloud-recent-days 0` should not keep image from 7/31/2018 as its age < 1 days, right?

Reply:
> Yeah, those sound right to me. However, since the calculation is done with whole days, depending on the time it was created on the day, it may or may not get picked up by the check. Personally I'm okay with that, since I'm running this twice a day and in the long run, I just want to keep "roughly" the number of days I'm specifying. How do you feel about the _date_ matching but the photo not being deleted because it's 25 hours old in some instances?

I like to keep days == 24hrs and use full timestamps for calcs. Using just dates requires proper TZ handling and can get tricky.

We can technically allow suffix for days and hrs, but that is extra complexity for just a narrow edge case that barely anybody faces.